config/default.yml

```diff
@@ -1,3 +1,7 @@
+Salsify/DandelionApplicationSerializer:
```

Comment:
I'll rename it `RailsApplicationSerializer` to be consistent with some of our other custom cops. We only get one-level of namespacing, so I've been trying to keep everything under `Salsify/`.